ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -128,6 +129,60 @@ class TreegraphSeries extends OrganizationSeries {
                 fillOpacity: 1,
                 states: {}
             },
+            link: {
+                /**
+                 * The color of the links between nodes.
+                 *
+                 * @type {Highcharts.ColorString}
+                 * @private
+                 */
+                color: Palette.neutralColor60,
+                /**
+                 * The line width of the links connecting nodes, in pixels.
+                 * @type {number} width in pixels
+                 * @default 1
+                 *
+                 * @sample   highcharts/series-organization/link-options
+                 *           Square links
+                 *
+                 * @private
+                 */
+                lineWidth: 1,
+                /**
+                 * Radius for the rounded corners of the links between nodes.
+                 * Works for `default` link type.
+                 *
+                 * @sample   highcharts/series-organization/link-options
+                 *           Square links
+                 *
+                 * @private
+                 */
+                radius: 10,
+                /**
+                 * Type of the link shape.
+                 *
+                 * @sample   highcharts/series-organization/different-link-types
+                 *           Different link types
+                 *
+                 * @type {'default' | 'curved' | 'straight'}
+                 * @default 'default'
+                 * @product highcharts
+                 *
+                 */
+                type: 'curved' as const
```

Comment:
Q: as const?

Reply:
I had some troubles with defining the types as string literals, and I found something like this:  typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

I thought it is useful. Is there some other way?

Reply:
(agreed solution: use const assertion to full object, not for every property inside)